src/global/BaseTest.java

```diff
@@ -206,4 +206,4 @@ public void restoreSystemInputOutput() {
         System.setIn(systemIn);
         System.setOut(systemOut);
     }
-}
+}
```

Comment:
Avoid changes to the framework files themselves

Reply:
It happenned upon merging main, I tried everything but it still views it as a change